src/Nest/Modules/Scripting/ExecutePainlessScript/ExecutePainlessScriptRequest.cs

```diff
@@ -0,0 +1,25 @@
+using System;
+using Newtonsoft.Json;
+
+namespace Nest
+{
+	public partial interface IExecutePainlessScriptRequest
+	{
+		[JsonProperty("script")]
+		IInlineScript Script { get; set; }
+	}
```

Comment:
missing `Context`? 

https://www.elastic.co/guide/en/elasticsearch/painless/current/painless-execute-api.html

Can be added later though, as there is only one context at this point

Reply:
Nevermind, just seen the commit message 😄